### PR TITLE
cfcli intro push

### PR DESCRIPTION
**Pull request by agracey**
Here's a initial intro to the cli and a sample node.js application

I also split up the pages and modified the generator so it can build multiple documents.

### CODE REVIEW

File: md/cfcli/apps.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Organizations and Spaces
+
+The applications and users of a Cloud Foundry platform are split into Organizations and Spaces. This gives a way to built a multi-tenant environment with minimal headache since each of these organizations and spaces can be given it's own resource quotas and access controls.
+
+As part of your free trial, you have administrator rights for your own organization with a few different spaces by default. By default, there are a few applications running in the "sample" space (selected at login).
```
Comment (timirnich) [inline md/cfcli/apps.md:5]: Do we maybe rather want this material to be independent of the exploration environment, i.e. no explicit references to our specific target environment?

File: md/cfcli/apps.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+To check which apps are running, run:
```
Comment (timirnich) [inline md/cfcli/apps.md]: From here we're not really talking about orgs and spaces anymore. Break this out into a "useful CLI commands" section?

**Review by timirnich — APPROVED** [review]
This is completely fine to go in - once we have initial content for all the parts we need we'll have to take a few steps back and look at the whole picture to see if we need to adjust something but let's focus on getting initial content in place - that will also enable us to get more input from others.

File: md/cfcli/apps.md (hunk 1 above)
Comment (agracey) [reply md/cfcli/apps.md:5]: I think we don't? There is other material that we can link to for more general usage. I do think we need to note what is "how we set it up" vs "cloud foundry in general" when there's discrepancy.

File: md/cfcli/apps.md (hunk 2 above)
Comment (agracey) [reply md/cfcli/apps.md]: Agreed. I started on one topic and then decided that a few other things flowed in the same area. I'll break it up into multiple files

File: md/cfcli/apps.md (hunk 1 above)
Comment (agracey) [reply md/cfcli/apps.md:5]: Added action item to trello to track this open question

**Pull request merged**